### PR TITLE
Fix #2269 : Added RTL support in Reading Display Size

### LINKED ISSUE #2269: [RTL] High-fi ReadingDisplaySize
`ReadingDisplaySize` needs RTL support as shown in below screen.
OP (Story Text Size) – RTL

Make sure that this works correctly in mobile (landscape + portrait) and tablet (landscape + portrait).

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by harsh-hash**
## Explanation

Fixes #2269 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.
Screenshot_1610300116
Screenshot_1610300163

### CODE REVIEW

**Comment by FareesHussain** [comment]
@harsh-hash please add screenshots in the description

**Comment by harsh-hash** [comment]
Screenshot_1610300116
Screenshot_1610300163

File: app/src/main/res/layout-land/reading_text_size_fragment.xml [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:layoutDirection="locale">
```
Comment (FareesHussain) [inline app/src/main/res/layout-land/reading_text_size_fragment.xml]: Revert this and add in the `line :72`

File: app/src/main/res/layout-sw600dp/reading_text_size_fragment.xml [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:layoutDirection="locale">
```
Comment (FareesHussain) [inline app/src/main/res/layout-sw600dp/reading_text_size_fragment.xml]: Revert this and add in the `line :42`

File: app/src/main/res/layout/reading_text_size_fragment.xml [hunk 3]
```diff
@@ -4,7 +4,8 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:layoutDirection="locale">
```
Comment (FareesHussain) [inline app/src/main/res/layout/reading_text_size_fragment.xml]: Revert this and add in the `line :66`

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@harsh-hash looks good but i recommend to add `layoutDirection="locale"`
only for the contraintLayout with seek bar

**Comment by FareesHussain** [comment]
Also update PR title to `Fix #2269: Added....`
and Add `Fixes #2269` in the Explanation

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@harsh-hash NIT changes requied this needs reformatting, I recommend you to use [macros](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros) for reformatting

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

**Comment by FareesHussain** [comment]
Tests need to be re-run

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Have you tried following these steps? https://github.com/oppia/oppia-android/wiki/RTL-Guidelines

For me on develop it does work correctly without any change.

Screenshot_1610379642

**Comment by FareesHussain** [comment]
@rt4914 Yes this works correctly in develop

**Comment by harsh-hash** [comment]
I have not sure about this issue, tried various things , and the first thing that worked, I thought that was the solution.

**Comment by FareesHussain** [comment]
> I have not sure about this issue, tried various things , and the first thing that worked, I thought that was the solution.

@harsh-hash thats great, We generally use layoutDirection="locale" to solve RTL related issues but as this was already fixed, it needs to be closed 

@rt4914 This can be closed

**Pull request closed without merging**